tests/_graphical_wait_login.pm

```diff
@@ -40,18 +40,22 @@ sub run {
     # desktop
     unless (get_var("DESKTOP") eq 'gnome' && get_var("INSTALL_NO_USER")) {
         unless (get_var("HDD_1") && !(get_var("PARTITIONING") eq "custom_resize_lvm")) {
-            # for Rocky Linux here happens to be a license acceptance screen
-            # the initial appearance can sometimes take really long
-            assert_screen "gdm_initial_setup_license", 120;
-            assert_and_click "gdm_initial_setup_license";
-            # Make sure the card has fully lifted until clicking on the buttons
-            wait_still_screen 5, 30;
-            assert_and_click "gdm_initial_setup_licence_accept";
-            assert_and_click "gdm_spoke_done";
-            # As well as coming back
-            wait_still_screen 5, 30;
-            assert_screen "gdm_initial_setup_license_accepted";
-            assert_and_click "gdm_initial_setup_spoke_forward";
+            # in 9.0+, license screens are not shown by default
+            # https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#enhancement_installer-and-image-creation
+            unless ($version eq '9.0') {
```

Comment:
The description above does not correspond to the actual conditional, it will work for now but not for 10.